hooks/post_gen_project.py

```diff
@@ -0,0 +1,41 @@
+import re
+import sys
+from pathlib import Path
+
+
+# >>> Optionally clearing comments comments
```

Comment:
typo - comments comments.
same thing at the end of the file

Reply:
thank you! fixed in c7d7b05